### PR TITLE
Java 14 Update CCE

### PR DESCRIPTION
**Pull request by taylorrassmannlibertyits**
Updated build files for Java 14 update.

### CODE REVIEW

File: community-care-eligibility-api/pom.xml [hunk 1]
```diff
@@ -4,11 +4,11 @@
   <parent>
     <groupId>gov.va.api.health</groupId>
     <artifactId>api-starter</artifactId>
-    <version>6.0.9</version>
+    <version>7.0.0</version>
```
Comment (jblefkove-va) [inline community-care-eligibility-api/pom.xml:7]: isn't the version 7.0.1?

File: community-care-eligibility-tests/pom.xml [hunk 2]
```diff
@@ -4,11 +4,11 @@
   <parent>
     <groupId>gov.va.api.health</groupId>
     <artifactId>test-starter</artifactId>
-    <version>6.0.9</version>
+    <version>7.0.0</version>
     <relativePath/>
   </parent>
   <artifactId>community-care-eligibility-tests</artifactId>
-  <version>2.0.51-SNAPSHOT</version>
+  <version>3.0.0-SNAPSHOT</version>
```
Comment (jblefkove-va) [inline community-care-eligibility-tests/pom.xml:11]: upgrade the mock-ee version below

File: community-care-eligibility/pom.xml [hunk 3]
```diff
@@ -4,11 +4,11 @@
...
-    <version>6.0.9</version>
+    <version>7.0.0</version>
     <relativePath/>
   </parent>
   <artifactId>community-care-eligibility</artifactId>
-  <version>2.0.51-SNAPSHOT</version>
+  <version>3.0.0-SNAPSHOT</version>
   <packaging>jar</packaging>
   <properties>
     <ee-client.version>3.0.6</ee-client.version>
```
Comment (jblefkove-va) [inline community-care-eligibility/pom.xml:14]: This is a project in https://github.com/department-of-veterans-affairs/health-apis-eligibility-and-enrollment-client . Upgrade that first.

File: developer.md [hunk 4]
```diff
@@ -1,7 +1,7 @@
 # Developer Guide
 
 ## Building
-- [Java Development Kit](https://openjdk.java.net/) 12
+- [Java Development Kit](https://openjdk.java.net/) 14
```
Comment (jblefkove-va) [inline developer.md:4]: The link should also be to Adopt OpenJDK now.

File: community-care-eligibility-api/pom.xml (hunk 1 above)
Comment (taylorrassmannlibertyits) [reply community-care-eligibility-api/pom.xml]: Yes, updated.

File: community-care-eligibility/pom.xml (hunk 3 above)
Comment (taylorrassmannlibertyits) [reply community-care-eligibility/pom.xml]: Updated for when eec gets merged.

File: developer.md (hunk 4 above)
Comment (taylorrassmannlibertyits) [reply developer.md]: Updated

File: community-care-eligibility-tests/pom.xml (hunk 2 above)
Comment (taylorrassmannlibertyits) [reply community-care-eligibility-tests/pom.xml:11]: Updated.

File: pom.xml [hunk 5]
```diff
@@ -4,7 +4,7 @@
   <parent>
     <groupId>gov.va.api.health</groupId>
     <artifactId>health-apis-parent</artifactId>
-    <version>6.0.9</version>
+    <version>7.0.1</version>
   </parent>
   <artifactId>community-care-eligibility-parent</artifactId>
   <version>2.0.51-SNAPSHOT</version>
```
Comment (jhulbert-va) [inline pom.xml:10]: Upgrade this.

**Pull request merged**